web/packages/teleterm/src/ui/Search/pickers/results.story.tsx

```diff
@@ -165,6 +166,82 @@ const SearchResultItems = () => {
         }),
       }),
     }),
+    makeResourceResult({
+      kind: 'app',
+      resource: makeApp({
```

Comment:
Could you make sure that each app shows a distinct and self-explanatory name, like dbs and servers do? At the moment, each app is called "foo".